base/docs/helpdb.jl

```diff
@@ -3792,7 +3792,34 @@ utf8(s)
 doc"""
     hvcat(rows::Tuple{Vararg{Int}}, values...)
 
-Horizontal and vertical concatenation in one call. This function is called for block matrix syntax. The first argument specifies the number of arguments to concatenate in each block row. For example, `[a b;c d e]` calls `hvcat((2,3),a,b,c,d,e)`.
+Horizontal and vertical concatenation in one call. This function is called for block matrix syntax. The first argument specifies the number of arguments to concatenate in each block row.
+
+```jldoctest
+julia> a, b, c, d, e, f = 1, 2, 3, 4, 5, 6
+(1,2,3,4,5,6)
+
+julia> [a b c; d e f]
+2x3 Array{Int64,2}:
+ 1  2  3
+ 4  5  6
+ 
```

Comment:
please remove the trailing whitespace from here and 3810, and if you've got a local source build it would also be handy if you could run `make && usr/bin/julia doc/genstdlib.jl` on this branch to propagate these changes into the RST.